### DISCUSSION TITLE
applying for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by GeorgeDPal** [community/community#58683] [GitHub Education]
### Hi everyone,

I want to apply for GitHub Student Developer pack. I have graduated on 2021 and currently I am learning new IT Skills. I love learning and also at the same time, I want to land a job in IT Industry.  Few times ago, **I came to know about GitHub student developer program. I wanted to apply for it. But since, I am not currently enrolled in university I couldn't apply for it. How can I avail the facility?** I want to enhance my skills to enrich.

#57916

**Comment by the-officialjosh** [comment]
The student pack is exclusively available for students, and once they have completed their schooling, the privileges associated with the pack are usually withdrawn. It's important to note that there might be a cost involved in accessing the pack post-graduation. However, there may be discounts or alternative pricing options available.
